lib/src/main/java/com/auth0/jwt/JWTCreator.java

```diff
@@ -327,11 +327,11 @@ private void addClaim(String name, Object value) {
     }
 
     private String sign() throws SignatureGenerationException {
-        String header = Base64.encodeBase64URLSafeString(headerJson.getBytes(StandardCharsets.UTF_8));
-        String payload = Base64.encodeBase64URLSafeString(payloadJson.getBytes(StandardCharsets.UTF_8));
+        String header = Base64.encodeBase64URLSafeString(headerJson.getBytes(Charset.forName("UTF-8")));
```

Comment:
Create an interface under the `com.auth0.jwt.interfaces` package to keep a constant and reference it everywhere instead of hardcoding the string value.